backend/tests/Squidex.Domain.Apps.Entities.Tests/Contents/DomainObject/Guards/GuardContentTests.cs

```diff
@@ -29,6 +30,7 @@ public class GuardContentTests : IClassFixture<TranslationsFixture>
         private readonly NamedId<DomainId> appId = NamedId.Of(DomainId.NewGuid(), "my-app");
         private readonly ClaimsPrincipal user = Mocks.FrontendUser();
         private readonly Instant dueTimeInPast = SystemClock.Instance.GetCurrentInstant().Minus(Duration.FromHours(1));
+        private readonly RefToken refToken = new RefToken("string", "string");
```

Comment:
I think refToken is called actor in most other tests

``
private readonly RefToken actor = new RefToken(RefTokenType.Subject, "123");
```